Update alert wording in Crons CLI (#6576)
getsentry/sentry-docs: src/docs/product/crons/getting-started/cli.mdx

<Alert level="note" title="Deprecation Notice">

> Starting from v2.16.1 of the Sentry CLI, the ability to monitor check-in authorization through an auth token has been deprecated. Read on to learn how to update your CLI and use your project's DSN instead.

Author: ```suggestion
Starting with v2.16.1 of the Sentry CLI, the ability to monitor check-ins using an auth token for authorization has been deprecated. Read on to learn how to update your CLI and use your project's DSN instead.
```
